README.md

```diff
@@ -156,6 +156,22 @@ all messages could be either duplicated (using `copyTo` attribute) or redirected
 
 While creating the FIFO queue, .fifo suffix will be added automatically to queue name.
 
+# Persisting queues configuration
+
+Queues configuration can be automatically persisted to external config file in typesafe config format.
+
+To make it work you need to pass implementation of `QueryPersister` trait while creating `SQSRestServer`. 
+To do it you can use `withQueuePersister` method from `SQSRestServerBuilder`.
+
+    SQSRestServerBuilder.withQueuePersister(QueueConfigStore(config))
+
+Additionally, you have to set `enabled` flag in `reference.conf` to true and specify `path` where the file 
```

Comment:
additionally? I would expect that when using the server specifying the flag would be enough. Alternatively (not additionally :) ), when using ElasticMQ in embedded mode, you don't really use the server config and can provide the persister by hand